build/vsts-ci.yml

```diff
@@ -296,6 +299,20 @@ extends:
           - script: $(Build.SourcesDirectory)/.dotnet/dotnet.exe build-server shutdown
             displayName: Dotnet Server Shutdown
 
+      - ${{ if eq(variables.enableSourceIndex, 'true') }}:
+        - template: /eng/common/templates-official/job/source-index-stage1.yml@self
+          parameters:
+            sourceIndexBuildCommand: ./build.cmd -ci -configuration Release /p:TargetArchitecture=x64 /bl
+            binlogPath: msbuild.binlog
+            presteps:
+            - task: NodeTool@0
```

Comment:
Why do we ne need Node.JS?  Maybe this was just something ASP.NET needed?

Reply:
I think you're right. I'll remove it